glean-core/src/metrics/labeled.rs

```diff
@@ -11,18 +11,11 @@ const MAX_LABELS: usize = 16;
 const OTHER_LABEL: &str = "__other__";
 const MAX_LABEL_LENGTH: usize = 61;
 
-/// Checks whether the given value matches the label regex.
+/// Checks whether the given label is sane. 
```

Comment:
Maaaybe this function also shouldn't be called `matches_label_regex` anymore, but more like `is_valid_label` or something like that?

Reply:
```suggestion
/// Checks whether the given label is sane.
```